wpilib/wpilib/command/subsystem.py

```diff
@@ -59,7 +59,7 @@ def initDefaultCommand(self):
         """
         pass
 
-    def periodic():
```

Comment:
You should use an editor with a linter to avoid these sorts of errors. :)

I use atom, with the extra linter-pyflakes package installed.